### DISCUSSION TITLE
Vercel limit exceeded, measures taken, still auto-blocked

### DISCUSSION THREAD

**Discussion by willemliufdmg** [vercel/vercel#6481] [General] [answered]
Hi,

Two days ago we've received an automated e-mail stating that our team has exceeded a limit and that we should take measures or our team will be blocked.
We've taken measures and replied to that e-mail to let you know that we did.
Yet two days after that mail (today). We received another mail saying our team has been blocked automatically.
And we can see in our dashboard that we have indeed been blocked.

I think that the 2-days between warning and block is too short. As it will surely not survive the weekends when, I assume, the people at Vercel are also having their free time.

And after a block there is no clear indication as to how to get unblocked again.

Regards,

Willem

**Comment by leo** [comment, marked as answer]
Thank you so much for sharing this!

Typically, we send out warning emails long before a block happens, and then, if you do decide to decrease your usage of the platform, no block will ever happen and no email about blocking will ever be sent out.

In your specific case, however, the block was already imminent because you've crossed the limits so far that there wouldn't be a way to recover that. In that case, the warning email is only sent out to let you know ahead of time that you will be blocked and give you time to, for example, upgrade your plan.

I do realize that, however, the information in the warning email that is making it seem like decreasing the usage could help, even in a scenario where a block is already imminent, is useless, and we're currently working on clarifying that.

**Reply by willemliufdmg** [reply]
Thank you for your answer. It clarifies a lot. It is indeed strange that the warning e-mail suggests that measures can be taken while the block is imminent.
As far as we ourselves could measure the traffic, we experienced a spike ~20 days ago which caused the exceeding of the limit. We thought we solved it by moving most of the static assets to our S3 buckets to only find ourselves receiving the warning ~20 days after the fact and a block 2 days after the warning.
It would also be nice to receive e-mails on different thresholds (50%, 75%, 90%, 100%). So we can actually take meaningful measures ahead of an imminent block.

**Reply by leo** [reply]
That's great feedback, thank you! We'll make sure of that. In the meantime, as an immediate fix, if you've received this email for the "Free" plan for Teams, I recommend upgrading to the "Pro" plan, as the "Free" plan is no longer part of our [pricing](https://vercel.com/pricing) and will disappear in the future (you will be notified about this separately).

If your use case is a personal one, however, you can also deploy your project on your Personal Account instead (Hobby plan), which is free forever.
